include/radix/core/gl/OpenGL.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef RADIX_CORE_GL_OPENGL_HPP
 #define RADIX_CORE_GL_OPENGL_HPP
 
+#ifdef _WIN32
+#define RADIX_GL_USE_GLAD
```

Comment:
Forcing GLAD on windows isn't the way to do it. Such detection should be in the root CMakeLists.